lms/templates/courseware_search/search_loading.underscore

```diff
@@ -0,0 +1 @@
+<%= gettext("Loading&hellip;") %>
```

Comment:
[We have removed ellipses from our platform.](https://github.com/edx/edx-platform/pull/6629) Please don't reintroduce them.